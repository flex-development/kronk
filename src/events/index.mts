/**
 * @file Entry Point - Events
 * @module kronk/events
 */

export { default as KronkEvent } from '#events/kronk.event'
export { default as OptionEvent } from '#events/option.event'
