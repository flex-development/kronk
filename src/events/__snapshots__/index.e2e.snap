// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:events > should expose public api 1`] = `
[
  "KronkEvent",
  "OptionEvent",
]
`;
